src/util/api/index.js

```diff
@@ -280,10 +280,11 @@ export const getLatestNFTClasses = () =>
 export const getTopNFTClasses = ({ before, after }) => {
   const qsPayload = {
     before,
-    after,
     ignore_list: LIKECOIN_NFT_API_WALLET,
   };
-  return `${LIKECOIN_CHAIN_API}/likechain/likenft/v1/ranking??${querystring.stringify(
+  if (after !== undefined) qsPayload.after = after;
+  if (before !== undefined) qsPayload.before = before;
```

Comment:
I actually wanted to remove both `before` and `after` from `const qsPayload `...
If they are removed from qsPayload then this will work? despite having them in function prototype